feat(curriculum): add Movie Ticket Booking Calculator workshop to Python cert (#64866)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-movie-ticket-booking-calculator/6958dcf685369045648d60bf.md

---
id: 6958dcf685369045648d60bf
title: Step 3
challengeType: 20
dashedName: step-3
---

# --description--

> Some of the informations can only be True or False. As you have learned in previous lessons, Booleans are used to store and truthy or falsy values.

Reviewer: ```suggestion
Some information can only be true or false. As you have learned in previous lessons, this can be represented using boolean values.
```
